ynput/ayon-photoshop
# Added webreview to default Settings

## Changelog Description
Without it review wouldnt show up in AYON

## Additional review information
PS still has separate `ExtractReview`. This PR is just matching current defaults from generic ExtractReview

## Testing notes:
1. publish something with reviewable from Photoshop
2. check webreview in AYON
